content/en/blog/falcosidekick-reponse-engine-part-6-cloud-run.md

```diff
@@ -0,0 +1,189 @@
+
+If you want to learn more about the `kubeconfig` files, you can follow this [link](https://ahmet.im/blog/mastering-kubeconfig/).
+
+- The second approach would be is producing a valid `~/.kube/config` with using library called `google.golang.org/api/` within the function code. Because the representation of the valid `~/.kube/config` file is [clientcmd/api/Config](https://pkg.go.dev/k8s.io/client-go@v0.21.1/tools/clientcmd/api#Config) in Go.
+
+We'll go through with the second approach in this demo. Thanks to Scott Blum, his [blog post](https://bionic.fullstory.com/connect-to-google-kubernetes-with-gcp-credentials-and-pure-golang/) is explaining this in more detail. I highly recommend you to check that out.
+
+Let's deploy the function. If you want to take a look at the function code, you can follow this GitHub [repository](github.com:developer-guy/kubernetes-response-engine-based-on-gke-and-cloud-run.git).
+
+One little note here is that we're going to use `ko` tool to build and push our container image which is created by Google. `ko` is a simple, fast container image builder for Go applications.
+
+If you want to learn more about it, you can follow the [official repository](https://github.com/google/ko) of the project.
+
+We are also going to use `Container Registry` as an image repository service provided by the Google Cloud to Store, manage, and secure your Docker container images, but this is not an only the option for this demo, you can use `DockerHub`, `quay.io`, etc.
+
+```shell
+$ git clone https://github.com/developer-guy/kubernetes-response-engine-based-on-gke-and-cloud-run.git
+$ cd kubernetes-response-engine-based-on-gke-and-cloud-run
+$ GKE_CLUSTER_NAME=$(kubectl config view --minify -o jsonpath='{.clusters[].name}{"\n"}')
+$ PROJECT_ID=$(gcloud config get-value project)
+$ FUNCTION_NAME=pod-deleter
+$ export KO_DOCKER_REPO=gcr.io/$PROJECT_ID # Please, change this variable if you are not using Container Registry.
+$ gcloud config set run/region us-west1
+$ gcloud config set run/platform managed
+$ gcloud run deploy $FUNCTION_NAME --image=$(ko publish .) \
+    --set-env-vars GKE_CLUSTER_NAME=$GKE_CLUSTER_NAME \
+    --set-env-vars PROJECT_ID=$PROJECT_ID
+...
+Allow unauthenticated invocations to [pod-deleter] (y/N)?  N
+
+Deploying container to Cloud Run service [pod-deleter] in project [developerguy-311909] region [us-west1]
+✓ Deploying... Done.
+  ✓ Creating Revision...
+  ✓ Routing traffic...
+Done.
+Service [pod-deleter] revision [pod-deleter-00002-cej] has been deployed and is serving 100 percent of traffic.
+Service URL: https://pod-deleter-uoz6q2wria-uw.a.run.app
+```
+
+### Install Falco + Falcosidekick
+
+Now, it is time to set up `Falco`, `Falcosidekick` with `Cloud Run` output type enabled.
```

Comment:
Now, it is time to set up `Falco` and `Falcosidekick` with the `Cloud Run` output type enabled.

Reply:
thank you so much, done.